### DISCUSSION TITLE
Missing mouse cursor in client

### DISCUSSION THREAD

**Discussion by ronny-nilsson** [any1/neatvnc#70] [Q&A] [answered]
Hi
Thanks for neatvnc!
I have a question about the cursor. My client don't get any... Despite running the "draw" example (painting work), I get no cursor. Effectively painting as a blind.

Can you give some tip here? I've tried both TigerVNC and Tight clients. Latest git master of neatvnc.

Thanks
/Ronny

**Comment by ronny-nilsson** [comment, marked as answer]
After further investigations I can answer my own question. There is a bug in the "draw" example. 	:grin:

**Reply by any1** [reply]
What's the bug?

**Reply by ronny-nilsson** [reply]
The call to `nvnc_set_cursor()` has swapped `width/height` with `hotspot_x/hotspot_y`.

**Reply by any1** [reply]
Fixed. Thanks!
